browser-based-ingestion/README.md

```diff
@@ -0,0 +1,249 @@
+### KVS Browser based Ingestion is used to stream video from the system's webcam to the KVS backend directly without any proxies. ###
```

Comment:
* `the system's webcam` -> `from your browser`, just so people understand this is a web based demo.

* `to the KVS backend directly without any proxies`, into a KVS Stream? Users will probably be confused on why this is called out. They just want it 'all to work' :)

Reply:
Heading could be as simple as Browser based ingestion and then you can have an intro section to explain what the focus of this demo is.

Reply:
Correct, I have made the changes